### PR TITLE
Translations Update, please integrate

### PR DESCRIPTION
**Pull request by rgeraghty**
please merge. cc @rgeraght

### CODE REVIEW

File: src/extension/_locales/de/messages.json [hunk 1]
```diff
@@ -630,6 +633,27 @@
   "user_switch": {
     "message": "Benutzer/in wechseln"
   },
+  "v7_context_menu": {
+    "message": "Installieren Sie die neue AEM Sidekick"
```
Comment (rofe) [inline src/extension/_locales/de/messages.json:637]: Use masculine when referring to the sidekick in german.

File: src/extension/_locales/de/messages.json [hunk 2]
```diff
@@ -630,6 +633,27 @@
...
+  },
+  "v7_hint_cover": {
+    "message": "https://www.aem.live/tools/sidekick/en/v7-teaser.png"
+  },
+  "v7_hint_description": {
+    "message": "Eine brandneue Version der AEM Sidekick-Erweiterung ist jetzt im Chrome Web Store verfügbar. Entscheiden Sie sich noch heute für verbesserte Leistung, neue Funktionen und ein nahtloses Authoring-Erlebnis."
+  },
+  "v7_hint_title": {
+    "message": "Die neue AEM Sidekick"
```
Comment (rofe) [inline src/extension/_locales/de/messages.json:646]: Use masculine when referring to the sidekick in german.

File: src/extension/_locales/de/messages.json (hunk 1 above)
Comment (rgeraghty) [reply src/extension/_locales/de/messages.json:637]: @rofe these should be improved once we get vendor human translation (to replace MT)

File: src/extension/_locales/de/messages.json (hunk 2 above)
Comment (rgeraghty) [reply src/extension/_locales/de/messages.json:646]: @rofe these should be improved once we get vendor human translation (to replace MT)

**Pull request merged**